### PR TITLE
Linter, Format and CI Check

### PR DESCRIPTION
**Pull request by louchebem06**

### CODE REVIEW

**Review by d-r-e — APPROVED** [review]

File: angular/src/app/header/header.component.html [hunk 1]
```diff
@@ -1,25 +1,37 @@
...
-			  <!-- A select of languages -->
-			  <select class="form-select bg-black text-white language-selector" (change)="changeLanguage($event)">
-				<option value="en" [selected]="currentLanguage == 'en'">English</option>
-				<option value="fr" [selected]="currentLanguage == 'fr'">Français</option>
-				<option value="es" [selected]="currentLanguage == 'es'">Español</option>
-			  </select>
+				<!-- A select of languages -->
+				<select
+					class="form-select bg-black text-white language-selector"
+					(change)="changeLanguage($event.target.value)">
```
Comment (axelcoezard) [inline angular/src/app/header/header.component.html:20]:
```suggestion
					(change)="changeLanguage($event?.target?.value)">
```

File: angular/src/app/path/block/block.component.html [hunk 2]
```diff
@@ -1,63 +1,89 @@
...
+				type="number"
+				class="form-control my-1"
+				[min]="80"
+				[max]="125"
+				[step]="1"
+				(click)="event($event)"
+				[(value)]="slider_mark"
+				color="green"
+				(keyup.enter)="show_slider = false"
+				(change)="setPlannedXP(project, $event.target.value)"
```
Comment (axelcoezard) [inline angular/src/app/path/block/block.component.html:85]:
```suggestion
				(change)="setPlannedXP(project, $event?.target?.value)"
```

**Review by axelcoezard — CHANGES_REQUESTED** [review]
Left a review

**Comment by d-r-e** [comment]
@louchebem06 could you please solve the conflicts? Thanks

**Pull request merged**